### PR TITLE
feat: Updating dependencies for Capacitor 5

### PR DESCRIPTION
**Pull request by theproducer**

### CODE REVIEW

File: assets/plugin-template/android/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.1'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```
Comment (markemer) [inline assets/plugin-template/android/build.gradle:14]: Is this the gradle version we want to pin? Or should it be one of the 7.4.1 ones?

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Added a few comments
Also gradle wrapper should be updated to 7.5 (the properties file and the actual wrapper files)
Also the plugin should switch to using namespace instead of package (like https://github.com/ionic-team/capacitor-plugins/pull/1503, might require code changes on the script that creates the plugin as it's probably writing the package name in the AndroidManifest.xml and should be switched to using a namespace)

File: src/template.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import type { OptionValues } from './options';
 
 const MUSTACHE_EXTENSION = '.mustache';
 
-export const CAPACITOR_VERSION = '^4.0.0';
+export const CAPACITOR_VERSION = '^5.0.0-alpha.1';
```
[the remark this answers is not in the document]
Comment (markemer) [reply src/template.ts:10]: Yeah, and in either case, we'll have to pin this again on release.

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Missed this previous comment:
> Also gradle wrapper should be updated to 7.5 (the properties file and the actual wrapper files)

**Review by giralte-ionic — APPROVED** [review]

**Review by IT-MikeS — APPROVED** [review]

**Review by giralte-ionic — APPROVED** [review]

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
enableJetifier should be removed from gradle.properties
https://github.com/ionic-team/create-capacitor-plugin/blob/main/assets/plugin-template/android/gradle.properties#L24

**Review by IT-MikeS — APPROVED** [review]

**Review by jcesarmobile — APPROVED** [review]

**Pull request merged**